src/AppConfig.ts

```diff
@@ -6,6 +6,10 @@ export const appConfig = {
   JWT_SECRET: get('appConfig.jwtSecret'),
   ACCESS_TOKEN_EXPIRE_MIN: get('appConfig.accessTokenExpireMin'),
   REFRESH_TOKEN_EXPIRE_MIN: get('appConfig.refreshTokenExpireMin'),
+  EMAIL_TOKEN_EXPIRE_DAY: get('appConfig.emailTokenExpireDay'),
```

Comment:
It's better to call it activationToken, not emailToken, because it's used only for activation, not for every email.